src/config/config.ts

```diff
@@ -172,6 +172,12 @@ const config = convict({
         format: String,
         default: "",
       },
+      mockAmplifyCreateDomainAssociationCalls: {
+        doc: "Mock createDomainAssociation calls to Amplify ",
+        env: "MOCK_AMPLIFY_CREATE_DOMAIN_ASSOCIATION_CALLS",
+        format: "required-boolean",
+        default: true,
+      },
```

Comment:
pls update staging/prod 1PW so we don't default to `true` :monkas:

Reply:
Yes, 1pw values are already updated!